resteasy/src/main/java/com/cerner/beadledom/resteasy/InjectorProcessor.java

```diff
@@ -0,0 +1,98 @@
+package com.cerner.beadledom.resteasy;
+
+import com.google.inject.Binding;
+import com.google.inject.Injector;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import javax.ws.rs.ext.Provider;
+import org.jboss.resteasy.plugins.guice.GuiceResourceFactory;
+import org.jboss.resteasy.spi.Registry;
+import org.jboss.resteasy.spi.ResourceFactory;
+import org.jboss.resteasy.spi.ResteasyProviderFactory;
+import org.jboss.resteasy.util.GetRestful;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Processor for registering JAX-RS providers and root resources from bindings tied to a Guice injector.
+ *
+ * @author Charan Panchagnula
+ * @since 3.3
+ */
+class InjectorProcessor {
+  private static final Logger logger = LoggerFactory.getLogger(InjectorProcessor.class);
+
+  private final Registry registry;
+  private final ResteasyProviderFactory providerFactory;
+
+  /**
+   * Creates a new instance for InjectorProcessor with thw given Resteasy resource registry
```

Comment:
"thw" -> "the"

Reply:
https://github.com/cerner/beadledom/pull/143/commits/bd6ab90bbf3a2dfb902bc7cda220fbd0a1cae674